tests/album-update-patch.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Album', () => {
+  let album;
+  beforeEach(async () => {
+    const { rows } = await db.query(
```

Comment:
similar to what I mentioned in the album-delete test file, add an artist first then an album and link them